src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/MigrationQueueService.kt

```diff
@@ -20,6 +20,8 @@ class MigrationQueueService(
   private val migrationQueue by lazy { hmppsQueueService.findByQueueId("migration") as HmppsQueue }
   private val migrationSqsClient by lazy { migrationQueue.sqsClient }
   private val migrationQueueUrl by lazy { migrationQueue.queueUrl }
+  private val migrationDLQSqsClient by lazy { migrationQueue.sqsDlqClient!! }
+  private val migrationDLQUrl by lazy { migrationQueue.dlqUrl!! }
 
```

Comment:
these are nullable in library - but shouldn't be null for this service - hence `!!`